app/controllers/orders/active_items.js

```diff
@@ -18,12 +27,51 @@ export default detail.extend({
     "sortProperties"
   ),
 
+  setDefaultSortingColumn: Ember.computed("dropDownItems", {
+    get(key) {
+      return ORDER_SORTING_OPTIONS[0];
+    },
+    set(key, value) {
+      return value;
+    }
```

Comment:
that's a setter that doesn't actually set anything, and the property's name `setXXX` makes it feel like a function.

it seems dependant on `dropDownItems` without every using that property which is weird.
Not sure what the intended behaviour is, but if you want a property that gets modified later, you can literally just do

```
sortingColumn: ORDER_SORTING_OPTIONS[0], // initial value

// and later

set('sortingColumn', column)
```

Reply:
Makes sense and its much cleaner.. Thanks.